package.json

```diff
@@ -5,7 +5,8 @@
   "main": "",
   "scripts": {
     "postinstall": "cd couch2pg && npm ci",
-    "test:e2e": "npm run test:e2e-data && npm run test:e2e-containers && mocha tests/**/*.spec.js --timeout 50000; npm run test:e2e-stop-containers ",
+    "test:e2e:local": "npm run test:e2e-data && npm run test:e2e-containers && mocha tests/**/*.spec.js --timeout 50000; npm run test:e2e-stop-containers",
+    "test:e2e": "npm run test:e2e-data && npm run test:e2e-containers && mocha tests/**/*.spec.js --timeout 50000 && npm run test:e2e-stop-containers ",
```

Comment:
why do we want to add the `test:e2e:local` test?  I see it only has `;` between the last two commands as opposed to `&&`.  I would think we would want the local tests to fail the same way the GH CI test would fail, yes?

Reply:
Yes, you’re right! Initially, I thought it might be useful for local testing to always ensure cleanup after the test, regardless of the outcome. However, it makes more sense to have consistent behavior between local tests and the CI tests. I’ll update it to reflect that.